web/src/components/bio.js

```diff
@@ -35,7 +35,7 @@ const Body = styled.div`
 `;
 
 const StyledImg = styled(Img)`
-    ${tw`h-auto max-w-full md:max-w-none md:col-start-10 md:h-96 md:w-96 md:mb-20 `}
+    ${tw`h-auto rounded-full max-w-full h-4/6 md:max-w-none md:col-start-10 md:h-96 md:w-96 md:mb-20 `}
```

Comment:
Please remove the `h-auto` class as it's being overwritten by the `h-4/6` class below the `md:` breakpoint.